<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1839</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyQt6 Layout Example</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_main">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_left">
      <item>
       <widget class="QLabel" name="labelLogo">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>80</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>LOGO</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonXplus">
        <property name="text">
         <string>X+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonXminus">
        <property name="text">
         <string>X-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonYplus">
        <property name="text">
         <string>Y+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonYminus">
        <property name="text">
         <string>Y-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonZplus">
        <property name="text">
         <string>Z+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonZminus">
        <property name="text">
         <string>Z-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelCamera">
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>603</width>
        <height>798</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>CAMERA WINDOW</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_right">
      <item>
       <widget class="QLabel" name="labelStatus">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>40</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>STATUS LABEL</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="markersLabel">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>Markers detected</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label1">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>LABEL 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="suctionOnButton">
        <property name="text">
         <string>Suction on</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="suctionOffButton">
        <property name="text">
         <string>Suction off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pickUpButton">
        <property name="text">
         <string>Pick up marker</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1839</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
